internal/system/agent/const.go

```diff
@@ -14,14 +14,25 @@
 package agent
 
 type ConfigurationStruct struct {
-	ReadMaxLimit        int
-	ValidateCheck       bool
-	AppOpenMsg          string
-	FormatSpecifier     string
-	ServicePort         int
-	ServiceTimeout      int
-	ServiceAddress      string
-	LoggingFile         string
-	LoggingRemoteURL    string
-	EnableRemoteLogging bool
+	ReadMaxLimit          int
+	ValidateCheck         bool
+	AppOpenMsg            string
+	FormatSpecifier       string
+	ServicePort           int
+	ServiceTimeout        int
+	ServiceAddress        string
+	LoggingFile           string
+	LoggingRemoteURL      string
+	EnableRemoteLogging   bool
+	OsLevelOperations     bool
+	DockerLevelOperations bool
+}
+
+type ManifestStruct struct {
+	EdgexServicesNotificationsName         string
+	EdgexServicesNotificationsHost         string
+	EdgexServicesNotificationsPort         int
+	EdgexServicesNotificationsProtocol     string
+	EdgexServicesNotificationsCommandstop  string
+	EdgexServicesNotificationsCommandstart string
```

Comment:
TOML comments above will affect this. In fact, if you take the recommendations above, You can add a property to the ConfigurationStruct.
`Manifests    map[string] config.ManifestInfo`

which would allow you access to the given manifest by its name or service key (as described above)